scripts/start-react-native.sh

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Just to bump up the modified files count eh?

Reply:
heading for a record here, got to  do what you got to do